llarp/router/router.cpp

```diff
@@ -655,16 +658,16 @@ namespace llarp
 
     const size_t connected = NumberOfConnectedRouters();
     const size_t N         = nodedb()->num_loaded();
-    if(N < minRequiredRouters)
+    if(N < 4)
```

Comment:
err...changed *from* a named amount to a hardcoded amount?

Reply:
fixed